app/javascript/src/components/editor/Editor.svelte

```diff
@@ -92,12 +92,13 @@
       const useNewlines = params.args_length >= $settings["autocomplete-min-parameter-newlines"]
 
       const apply = v.args.map(a => {
-        let string = a.default?.toString().toLowerCase().replaceAll(",", "")
-        if (useParameterObject) string = `${ useNewlines ? "\n\t" : "" }${ a.name }: ${ string }`
+        const name = toCapitalize(a.name?.toString().toLowerCase())
+        let defaultValue = a.default?.toString().toLowerCase().replaceAll(",", "")
+        defaultValue = lowercaseDefaults.includes(defaultValue.toLowerCase()) ?
+          defaults[toCapitalize(defaultValue)] :
+          toCapitalize(defaultValue)
```

Comment:
This is a bit hard to read, what do you thing of something like this?

```suggestion
        let defaultValue = a.default?.toString().toLowerCase().replaceAll(",", "")
        
        if (lowercaseDefaults.includes(defaultValue.toLowerCase()) defaultValue = defaults[toCapitalize(defaultValue)]
        else defaultValue =toCapitalize(defaultValue)
```